glusterd2/commands/volumes/volume-start.go

```diff
@@ -69,6 +73,12 @@ func volumeStartHandler(w http.ResponseWriter, r *http.Request) {
 	logger := gdctx.GetReqLogger(ctx)
 	volname := mux.Vars(r)["volname"]
 
+	query := r.URL.Query().Get("force")
```

Comment:
Name this variable as `force`. There can be many query params.

Reply:
done